components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt

```diff
@@ -320,13 +337,22 @@ class Nimbus(
 
     @WorkerThread
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    private fun setExperimentsLocallyOnThisThread(payload: String) {
```

Comment:
Looks like the caller https://github.com/mozilla-mobile/android-components/blob/cbc512c1fa61d2e43589fc59decc1167d8b4d130/components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt#L325 does IO and needs to catch to?